crates/channel/src/channel_store.rs

```diff
@@ -19,7 +19,7 @@ use settings::Settings;
 use std::{mem, sync::Arc, time::Duration};
 use util::{ResultExt, maybe};
 
-pub const RECONNECT_TIMEOUT: Duration = Duration::from_secs(30);
+pub const RECONNECT_TIMEOUT: Duration = Duration::from_secs(1);
```

Comment:
Whoops. This is here just because I wanted to get into disconnected state quicker, but we should definitely not go forward with it.
```suggestion
pub const RECONNECT_TIMEOUT: Duration = Duration::from_secs(30);
```